### DISCUSSION TITLE
“快速收起评论”相关

### DISCUSSION THREAD

**Discussion by Neo-Ze** [the1812/Bilibili-Evolved#3485] [提问 / Q&A] [answered]
目前“快速收起评论”功能是在本人动态页面生效的，请问能否在UP主空间内浏览动态时也生效呢？
![image](https://user-images.githubusercontent.com/81399833/177552768-c05374ea-839d-436f-b0fa-ab1790906e2a.png)

**Comment by Geeyun-JY3** [comment, marked as answer]
是 bug，又失效了
